testdata/picard/reference/csd_dict.dict

```diff
@@ -0,0 +1,9 @@
+@HD	VN:1.5
+@SQ	SN:chr1	LN:101	M5:bd01f7e11515bb6beda8f7257902aa67	UR:file:/home/pavel/git/epm-cmbi/picard-pho/testdata/picard/reference/test.fasta
```

Comment:
please remove references to personal location of files from repository (since it doesn't matter, replace with dummy path)